### DISCUSSION TITLE
Extras repositories and Linux Mint

### DISCUSSION THREAD

**Discussion by moonshaadow** [falkTX/Carla#1384] [Q&A]
Are the Ubuntu 18.04 and 20.04 specific extras good to apply on Linux Mint 20.1 ?

**Comment by falkTX** [comment]
how is this a question for carla?

but to answer, depends if that mint is based on 18.04 or 20.04, obviously. you need to find that yourself, I dont know.

**Reply by moonshaadow** [reply]
Yes I'm sorry. It's not related to Carla.

**Reply by v1993** [reply]
Linux Mint 20.1 is based on Ubuntu 20.04 as far I know, so you should be good.
